Item.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refrigerator_EX
+{
+    class Item
+    {
+        private static int nextID = 1;
```

Comment:
naming convention wrong, missing `_` and using this way is not thread safe, should use GUID instead.